docs/contributing/version-control.md

```diff
@@ -60,7 +60,8 @@ Other considerations:
 Zulip expects you to structure the commits in your pull requests to form
 a clean history before we will merge them.  It's best to write your
 commits following these guidelines in the first place, but if you don't,
-you can always fix your history using `git rebase -i`.
+you can always fix your history using `git rebase -i` (more on that
+[here](../git/fixing-commits.md#fixing-commits)).
```

Comment:
This is a broken link, you need to link via `.html#...` when linking to an anchor with current Sphinx.

Reply:
Hmm, I can do without the #fixing-commits; I included it earlier when I ran into a different error (turns out I'd left out an extension altogether) because of https://github.com/zulip/zulip/pull/13668#issuecomment-574495945, but it seems that issue only affects external links. I'm trying again with just `../git/fixing-commits.md`, to be as consistent as possible with links elsewhere.

Reply:
And it's working, with `../git/fixing-commits.md`.

Reply:
Just for background, due to bugs in the ReadTheDocs system, we have two styles for how to write links, depending on whether there's an anchor.  Once the bug is fixed, we expect the syntax you wrote above to work, but until then, it's either linking to `.md` or `.html#foo`.

Reply:
Makes sense, thank you. 🙂 Again, omitting `#foo` was preferable in the first place, for this case; I only added it because I was aware there was a bug, from https://github.com/zulip/zulip/pull/13668#issuecomment-574495945. In that case, using `.md#foo` was a workaround, but, it turns out, only for external links. So, for internal links, it sounds like I can use `.html#foo` if I need an anchor, and `.md` if not.